src/electionguard-ui/ElectionGuard.UI.Lib/Models/BallotRecord.cs

```diff
@@ -26,4 +26,9 @@ public partial class BallotRecord : DatabaseRecord
     public BallotRecord() : base(nameof(BallotRecord))
     {
     }
+
+    public override string ToString()

```

Comment:
could we copy the 2 lines from the bottom of the ContextRecord for this for consistancy?